src/components/App/App.js

```diff
@@ -6,37 +6,37 @@ import { setGameState } from '../../actions/gameState';
 import styles from './App.css';
 import Header from '../Header/Header';
 import Phrase from '../Phrase/Phrase';
-import Alphabet from '../Alphabet/Alphabet';
+import SwitchKeyboard from '../Keyboard/SwitchKeyboard/SwitchKeyboard';
 import ShowGallows from '../ShowGallows/ShowGallows';
 import GameOver from '../GameOver/GameOver';
```

Comment:
Tip of the day :wink:
[You can avoid repeating writing name of a class twice, thanks to ability to create index.js file in the containing folder.
e. g. If you've class `GameOver` is in folder named `GameOver/`, you can add file `index.js` under `GameOver/index.js`;
 In `index.js` you would've `export * from './GameOver` which exports all insides of `GameOver` outside of a folder,
and then during import you can just write `import GameOver from `../GameOver`](https://alligator.io/react/index-js-public-interfaces/)

Reply:
I didn't realize that duplicate name in import is so much inconveniente that there is a necessity to create new files index.js only for exports.
But then there is an issue with ESLint:
- in index.js files it says 
`export * from `**'./GameOver'**`;`
> No named exports found in module './GameOver'.eslint(import/export)
- in components it says
`import `**NewGame**` from '../NewGame';`
> No default export found in module.eslint(import/default)

Bold part is that which ESLint is referring to.